src/adapt/layout.js

```diff
@@ -1526,7 +1607,17 @@ adapt.layout.Column.prototype.layoutBreakableBlock = function(nodeContext) {
         // TODO: should this be done after first-line calculation?
         var edge = self.calculateEdge(resNodeContext, checkPoints, checkPointIndex,
             checkPoints[checkPointIndex].boxOffset);
-        var overflown = self.isOverflown(edge);
+        var overflown = false;
+        if (!adapt.layout.isOrphan(resNodeContext.viewNode)) {
```

Comment:
`resNodeContext` can be null when an inline element is flown into a partition by `-epubx-flow-into`.
Example: [null-resNodeContext.html](https://gist.github.com/kwkbtr/35948024993927cad02f309ee28bbe9b)